code/game/response_team.dm

```diff
@@ -508,6 +508,7 @@ var/ert_request_answered = 0
 	suit = /obj/item/clothing/suit/space/hardsuit/ert/engineer
 	suit_store = /obj/item/weapon/tank/emergency_oxygen/engi
 	glasses = /obj/item/clothing/glasses/meson
+	belt = /obj/item/weapon/storage/belt/utility/response_team/full/New()
```

Comment:
@giblinator1337 Travis is failing because of the New() statement here. 
`/obj/item/weapon/storage/belt/utility/response_team/full` should suffice.